Occasionally error: The WASM interface has not been initialized

See https://github.com/litentry/litentry-parachain/issues/1364

In our recent CIs, parachain-launch gives out errors when generating the relaychain genesis files, stack trace:

```
Trace: Error: Error: The WASM interface has not been initialized. Ensure that you wait for the initialization Promise with waitReady() from @polkadot/wasm-crypto (or cryptoWaitReady() from @polkadot/util-crypto) before attempting to use WASM-only interfaces.
    at /home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@polkadot/wasm-crypto/cjs/bundle.js:46:13
    at sr25519PairFromSeed (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@polkadot/util-crypto/cjs/sr25519/pair/fromSeed.js:22:82)
    at Object.sr25519 (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@polkadot/keyring/cjs/keyring.js:19:56)
    at Keyring.createFromUri (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@polkadot/keyring/cjs/keyring.js:256:115)
    at generateRelaychainGenesisFile (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@open-web3/parachain-launch/lib/index.js:1[43](https://github.com/litentry/litentry-parachain/actions/runs/4250721399/jobs/7393584955#step:8:44):31)
    at generate (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@open-web3/parachain-launch/lib/index.js:374:5)
    at fatal (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@open-web3/parachain-launch/lib/index.js:55:13)
make: *** [Makefile:175: generate-docker-compose-rococo] Error 1
```

Strangely it only happens sometimes, and only recently, maybe have you got a quick idea what's happening? Version/dependency problem? 

We have been using the latest parachain-launch (1.2.2) for a long time, and [this](https://github.com/litentry/litentry-parachain/blob/dev/docker/rococo-parachain-launch-config.yml) is our parachain-launch config.
Thanks for any help!

--- a/src/index.ts
+++ b/src/index.ts
@@ -5,6 +5,7 @@ import readline from 'readline-sync';
 import shell from 'shelljs';
 import { Keyring } from '@polkadot/api';
 import { cryptoWaitReady, encodeAddress, decodeAddress } from '@polkadot/util-crypto';
+import { waitReady } from '@polkadot/wasm-crypto';
 import yargs from 'yargs';
 import { hideBin } from 'yargs/helpers';
 import _ from 'lodash';
@@ -423,6 +424,7 @@ const generateDockerfiles = (config: Config, output: string, yes: boolean) => {
  */
 const generate = async (config: Config, { output, yes }: { output: string; yes: boolean }) => {
   await cryptoWaitReady();
+  await waitReady();
 
   if (!config?.relaychain?.chain) {
     return fatal('Missing relaychain.chain');

